### PR TITLE
Add Tailwind Size Conversion extension

### PR DESCRIPTION
**Pull request by kemiljk**
## Description

Converts pixels to Tailwind class values and back. This is based on a fork of [Px to rem converter](https://github.com/raycast/extensions/tree/main/extensions/px-to-rem-converter).

## Screencast

Screenshot - DuckDuckGo - Add px-to-tailwind extension by kemiljk · Pull Request … - 001190

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/px-to-tailwind/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "px-to-tailwind",
+  "title": "Px to Tailwind",
```
Comment (pernielsentikaer) [inline extensions/px-to-tailwind/package.json:4]:
```suggestion
  "title": "Tailwind Size Conversion",
```
Could this be a better naming?

File: extensions/px-to-tailwind/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "description": "Converts pixels to Tailwind class values",
```
Comment (pernielsentikaer) [inline extensions/px-to-tailwind/package.json:5]:
```suggestion
  "description": "Converts pixels to Tailwind class values and vice versa",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- What about copying `--tw-6` to the clipboard rather than just 6 (Maybe for them both)
 
px-to-tailwind 2023-06-06 at 09 50 45

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
Also a [metadata image](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) would be nice

**Comment by kemiljk** [comment]
Thanks @pernielsentikaer! I’ve made all of the tweaks except the tailwind class copying. Because a Tailwind class prefix can be one of many things, and never actually `—tw-` I thought it was best to just copy the arbitrary value. Showing `—tw-` in the UI is more a way to show that it would be prefixed with a Tailwind class and mirrors how they appear when you view the inspector on a web project.

**Review by kemiljk — COMMENTED** [review]
Changes should be all addressed!

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**